server/package.json

```diff
@@ -14,13 +14,13 @@
   ],
   "scripts": {
     "build": "cd .. && cd client && npm run build && cd .. && cd server",
-    "start": "ASSET_COMPUTE_DEV_TOOL_ENV=development concurrently \"nodemon bin/www\" \"npm run client\"",
-    "server": "ASSET_COMPUTE_DEV_TOOL_ENV=production node bin/www",
-    "client": "node set-port.js && cd ..&& cd client && npm start",
-    "pretest": "eslint src set-port.js test routes",
+    "start-dev": "ASSET_COMPUTE_DEV_TOOL_ENV=development concurrently \"nodemon index\" \"npm run client\"",
+    "start-prod": "ASSET_COMPUTE_DEV_TOOL_ENV=production node index",
```

Comment:
this does nothing since `index.js` exports `start`,

try something like this:
```
node -e 'require("./index.js").start()
```

or use a library like [make-runnable](https://www.npmjs.com/package/make-runnable) to be able to call `start` by a command like:
```
node index start
```